src/Pact/Eval.hs

```diff
@@ -701,12 +704,22 @@ prepareUserAppArgs Def{..} args i = do
   return (as',ft')
 
 -- | Instantiate args in body and evaluate using supplied action.
-evalUserAppBody :: Def Ref -> ([Term Name], FunType (Term Name)) -> Info -> Gas
-                -> (Term Ref -> Eval e a) -> Eval e a
-evalUserAppBody Def{..} (as',ft') ai g run =
-  let bod' = instantiate (resolveArg ai (map mkDirect as')) _dDefBody
-      fa = FunApp _dInfo (asString _dDefName) (Just _dModule) _dDefType (funTypes ft') (_mDocs _dMeta)
-  in appCall fa ai as' $ fmap (g,) $ run bod'
+evalUserAppBody
```

Comment:
Admittedly a legibility improvement but appears to offer no change in functionality and makes review burdensome to see if anything changed. Fine for now but please be restrained about this in general.